fiat-api/src/test/groovy/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluatorSpec.groovy

```diff
@@ -198,10 +190,32 @@ class FiatPermissionEvaluatorSpec extends Specification {
   }
 
   @Unroll
-  def "should allow access to unknown applications"() {
-    given:
-    def authentication = new PreAuthenticatedAuthenticationToken("testUser", null, [])
+  def "should deny access to an application that has an empty set of authorizations"() {
```

Comment:
Just asserting that we're a-ok with resources that exist on a `UserPermission` but don't themselves have any authorizations.